Packages/io.chainsafe.web3-unity/Samples~/Web3.Unity/Scripts/Scenes/Login.cs

```diff
@@ -137,11 +122,11 @@ private async void TryAutoLogin()
                 .Configure(ConfigureCommonServices)
                 .Configure(services =>
                 {
-                    services.UseWebPageWallet(
+                    services.UseWalletConnectWallet(
                         new WebPageWalletConfig
                         {
                             SavedUserAddress = savedAccount,
-                            WalletConnectConfig = BuildConfig()
+                            WalletConnectConfig = BuildWalletConnectConfig()
```

Comment:
Can we merge WalletConnectConfig and WebPageWalletConfig (this one should be renamed too) in one class?